core/templates/dev/head/components/CkEditorWidgetsInitializer.js

```diff
@@ -131,7 +129,7 @@ oppia.run([
             downcast: function(element) {
               // Clear the angular rendering content, which we don't
               // want in the output.
-              element.children[0].setHtml('');
+              (<CKEDITOR.htmlParser.element>element.children[0]).setHtml('');
```

Comment:
The angle bracket notation is used for typecasting. The setHtml property does not exits on type of node (children[0] is of type node) as per CKEditor definitions. So, we need to typecast it to element.

I will add a comment for the same.

Reply:
Actually, maybe instead of comments, we should probably add it to the style guide on the wiki instead (there actually ended up being quite a few more instances than I expected). Could we add precise instructions on when to use it and when not to?

Reply:
Yes, sure that would be better because there are many places where we need typecasting.

Reply:
Filed #6443 to keep a track.